starboard/rdk/launcher.py

```diff
@@ -0,0 +1,426 @@
+#
+# Copyright 2018 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""RDK implementation of Starboard launcher abstraction."""
+
+import functools
+import logging
+import os
+import re
+import signal
+import six
+import sys
+import threading
+import time
+import contextlib
+
+import pexpect
```

Comment:
Is pexpect something we have to use? @kaidokert I know you were looking at some alternative solutions, it seems like those could be more easily implemented on a new platform like this?

Reply:
Looks like we have an unfulfilled MH depenency